sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -537,6 +542,24 @@ object Radar {
             override fun onComplete(config: RadarConfig) {
                 locationManager.updateTrackingFromMeta(config?.meta)
                 RadarSettings.setFeatureSettings(context, config?.meta.featureSettings)
+
+                if (config.googlePlayProjectNumber != null) {
+                    // Create an instance of a standard integrity manager
+                    val standardIntegrityManager = IntegrityManagerFactory.createStandard(context)
+                    standardIntegrityManager.prepareIntegrityToken(
+                        StandardIntegrityManager.PrepareIntegrityTokenRequest.builder()
+                            .setCloudProjectNumber(config.googlePlayProjectNumber)
+                            .build()
+                    )
+                        .addOnSuccessListener { response ->
+                            Radar.standardIntegrityTokenProvider = response
+                            logger.d("successful warm up of the integrity token provider")
```

Comment:
similar to above ^ thinking about any potential race condition with the onSuccessListener if initialize is called twice

Reply:
nit: can we also call `response` here `tokenProvider` for future ref of what the response is (a la https://developer.android.com/google/play/integrity/standard#prepare-integrity)

Reply:
I couldn't produce a race condition issue when testing.